.eslintrc

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
It doesn't appear that you went through and did a final code review for ESLint. That should be done.